test/ui-test/testSuites/suite_status/shared/steps/chatSteps.py

```diff
@@ -73,6 +73,15 @@ def step(context, message):
 def step(context, message_index, message):
     _statusChat.reply_to_message_at_index(message_index, message)
     _statusChat.verify_last_message_sent(message)
+    
+@When("the user can edit the message at index |any| and change text to \"|any|\"" )
```

Comment:
This line is a bit confusing. Maybe it should read
```suggestion
@When("When the user edits the message at  index |any| and changes it to \"|any|\"" )
```